app/api/edlinker/messages.rb

```diff
@@ -4,9 +4,19 @@ class Edlinker::Messages < Grape::API
   namespace :users do
     desc 'create message for student'
     params { use :message_params }
-    post ':user_id/messages' do
-      current_student
-      @current_user.messages.create(params[:message])
+    post ':student_id/messages' do
```

Comment:
as we already talked on a meeting u need to remove student, teacher and user ids from urls